concourse/scripts/pipecleaner.py

```diff
@@ -40,6 +44,27 @@ def load_pipeline(self, filename):
         raw = re.sub('\{\{.*?\}\}', 'DUMMY', raw)
         return yaml.load(raw)
 
+    def shellcheck(self, shell, job, task, args):
+        if not self.shellcheck_enabled:
+            return
+
+        if args[-2] != '-c':
+            print "Odd args " + job + "." + task
+            print args
+            return
```

Comment:
Possibly;  it's only in a few places, so the consequence of merging this might be to un-weird it.  It just broke when I was hacking.

```
Odd args generate-secrets.generate-cf-secrets
Odd args generate-bosh-config.extract_terraform_outputs
Odd args bosh-cli.run-bosh-cli
Odd args init-bucket.try-fetch-bucket-state
```